detection-rules/attachment_encrypted_pdf_cred_theft.yml

```
@@ -23,6 +23,24 @@ source: |
     or any(ml.nlu_classifier(beta.ocr(beta.message_screenshot()).text).intents,
            .name == "cred_theft" and .confidence in ("medium", "high")
     )
+    or (
+      (
+        regex.icontains(body.current_thread.text,
+                        'PDF(\s((Access|Unlock|Decrypt)\s?Code|Passcode))'
```

```suggestion
                        'PDF\s*(?:Access|Unlock|Decrypt)\s*(?:Pass)?code'
```
PDF (Access OR Unlock OR Decrypt) (Passcode OR code)

---

looks good to me!